Updates for NextCloud integration

adavenpo opened the pull request.

edwinsteele commented on ansible/inventory:
```diff
@@ -1 +1 @@
-192.168.20.183 ansible_user=pi
+192.168.1.244 ansible_user=pi
```
People are going to change this based on their system - no single IP address will be correct - perhaps we should just leave it as an empty file (with a comment showing format). Thoughts?

edwinsteele commented on ansible/roles/bootstrap/tasks/main.yml:
```diff
@@ -1,7 +1,7 @@
 - name: Add debian signing keys, necessary for backports repo
   apt_key:
-    keyserver: keyserver.ubuntu.com
+    keyserver: hkp://keyserver.ubuntu.com:80
```
Thanks for this. Didn't consider corporate firewalls!

edwinsteele commented on ansible/roles/webserver-content/tasks/main.yml:
```diff
@@ -70,3 +82,33 @@
+    #force=yes
+
+- name: Pull python packages
+  shell: "pip install webapp2 webob"
```
Please use the [pip task](http://docs.ansible.com/ansible/pip_module.html). Pretty sure that that the python pip package is not installed by default (not sure what's it's called in debian), so you'll need to add that too using apt.

edwinsteele commented on ansible/roles/webserver-content/tasks/main.yml:
```diff
@@ -70,3 +82,33 @@
+  shell: "cp -a /tmp/BibleBox-nextcloud/wsgi /var/www/wsgi"
+
+- name: Add app link to uwsgi config
+  shell: "ln -fs /var/www/wsgi/app.ini /etc/uwsgi/apps-enabled/app.ini"
```
Please use the [file task](http://docs.ansible.com/ansible/file_module.html) to create the symlink.

adavenpo commented:
Thanks, Edwin...I think this addresses your comments.  FWIW, I think it's worth leaving at least an example line in the inventory for those unfamiliar with ansible.

edwinsteele commented:
Thanks Andrew. Much appreciated.
